packages/utils/src/utils/gateway.ts

```diff
@@ -26,3 +27,28 @@ export const getExplorerLink = (
 
   return { href, title }
 }
+export const signTxServiceMessage = async (
```

Comment:
Nit: this is not a generic function for signing Transaction Service messages.
```suggestion
export const signTxDeletionMessage = async (
```